swift/ql/test/library-tests/regex/parse.expected

```diff
@@ -6492,112 +6492,117 @@ regex.swift:
 
 #  206| [RegExpNamedCharacterProperty] [:aaaaa:]
 
-#  210| [RegExpGroup] (?i)
+#  211| [RegExpGroup] (?i)
 #-----| 0 -> [RegExpConstant, RegExpNormalChar] i
 
-#  210| [RegExpSequence] (?i)abc
+#  211| [RegExpSequence] (?i)abc
 #-----| 0 -> [RegExpGroup] (?i)
 #-----| 1 -> [RegExpConstant, RegExpNormalChar] abc
 
-#  210| [RegExpConstant, RegExpNormalChar] i
+#  211| [RegExpConstant, RegExpNormalChar] i
 
-#  210| [RegExpConstant, RegExpNormalChar] abc
+#  211| [RegExpConstant, RegExpNormalChar] abc
 
-#  211| [RegExpGroup] (?s)
+#  212| [RegExpGroup] (?s)
 #-----| 0 -> [RegExpConstant, RegExpNormalChar] s
 
-#  211| [RegExpSequence] (?s)abc
+#  212| [RegExpSequence] (?s)abc
 #-----| 0 -> [RegExpGroup] (?s)
 #-----| 1 -> [RegExpConstant, RegExpNormalChar] abc
 
-#  211| [RegExpConstant, RegExpNormalChar] s
+#  212| [RegExpConstant, RegExpNormalChar] s
 
-#  211| [RegExpConstant, RegExpNormalChar] abc
+#  212| [RegExpConstant, RegExpNormalChar] abc
 
-#  212| [RegExpGroup] (?is)
+#  213| [RegExpGroup] (?is)
 #-----| 0 -> [RegExpConstant, RegExpNormalChar] is
 
-#  212| [RegExpSequence] (?is)abc
+#  213| [RegExpSequence] (?is)abc
 #-----| 0 -> [RegExpGroup] (?is)
 #-----| 1 -> [RegExpConstant, RegExpNormalChar] abc
 
-#  212| [RegExpConstant, RegExpNormalChar] is
+#  213| [RegExpConstant, RegExpNormalChar] is
 
-#  212| [RegExpConstant, RegExpNormalChar] abc
+#  213| [RegExpConstant, RegExpNormalChar] abc
 
-#  214| [RegExpConstant, RegExpNormalChar] abc
+#  214| [RegExpGroup] (?i-s)
```

Comment:
GitHub's diff view isn't particularly useful here, but I *think* the changes look right.

Reply:
I wouldn't read too much into the changes to `parse.expected` in this PR - I think all of the new test cases had trivial regexps such as `"abc"` (i.e. the parse is uninteresting), and the rest of the diff is just stuff that moved to a new line number (such as the `"(?i-s)abc"` case you've highlighted - it was line 227 previously).